.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,157 @@
+- **Explain why this enhancement would be useful** to most Reasonable Performance Computing SIG's Website & Knowledgebase users. You may also want to point out the other projects that solved it better and which could serve as inspiration.
+
+<!-- You might want to create an issue template for enhancement suggestions that can be used as a guide and that defines the structure of the information to be included. If you do so, reference it here in the description. -->
+
+### Your First Guide Contribution
+
+Guides live under the `optimisations/` and `profilers/` directories respectively, grouped by language. Each guide is written in Markdown (`.md`) and should:
+
+- Setup the YAML header:
+  - `level`, The priority of the guide in search results, make a judgement call, more common issues should have a level closer to 1.
+  - `published`, By default this should be set to `true`.
+  - `authors`, The name of author/s that have contributed to the guide.
+  - `name`, A short and general name which describes the optimisation or the profiler's name.
+  - `language`, An array of languages that guide refers to. *Be careful to ensure this matches the capitalisation of other guides for the language!*
+  - `subcategory`, *(Optimisations only)* Ways to sub-categorise the optimisation, e.g. the library it corresponds to.
+  - `tags`, *(Optimisations only)* An array of terms which may help this guide be found when searching (with the website's crude JS search).
+  - `style`, *(Profiler's only)* The profiling styles the profiler supports (e.g., Function-Level, Line-Level, Memory, Timeline, Hardware-Metrics, Benchmarking)
+  - `website`, *(Profiler's only)* URL of the profiler's main website.
+- Each guide should begin with a short paragraph to introduce the guide
+  - This will also appear in the directory, before users click through to the guide
+  - The first paragraph should be followed by `<!--more-->`, which tells Jekyll where to place the fold.
+- Within the guide provide concise examples, code snippets, screenshots and references where applicable.
+  - Optimisation guides can really benefit from a small usable example that readers can test and modify.
+- Use headings (`##`, `###`, etc.) to organise sections.
+- Keep the tone educational and practical.
+
+You can use existing guides as templates—for example, [`_optimisations/python/numba.md`](https://github.com/sig-rpc/sig-rpc.github.io/blob/master/_optimisations/python/numba.md) or [`_profilers/python/cprofile.md`](https://github.com/sig-rpc/sig-rpc.github.io/blob/master/_profilers/python/cprofile.md).
+
+Once written, open a pull request against the `master` branch.
```

Comment:
Are we still using `master`? `main` has been the default for 5+ years and, at least for me, is pretty deeply ingrained into my muscle memory by now.